client/views/home.js

```diff
@@ -34,7 +35,6 @@ module.exports = function () {
     init()
   }
 
-  var torrent
```

Comment:
Simply changing this to `let` did not suffice because, due to `init()` being called above, accessing `torrent` in `init()` would throw a `ReferenceError`. Instead, I’ve moved this declaration to the top of the function.

Reply:
Thanks for explaining!